src/object/schema_manager.c

```diff
@@ -3385,6 +3385,26 @@ sm_check_reuse_oid_class (MOP op)
   return false;
 }
 
+/*
+ * sm_is_replication_class() - tests the replication class flag of a class object.
+ * return: true if replication is enabled, false otherwise
+ * op(in): class object
+ */
+bool
+sm_is_replication_class (MOP op)
```

Comment:
```suggestion
sm_is_replicable_class (MOP op)
```
복제 가능한 클래스인지 확인하는 것 같아, 이름 변경 제안 드립니다.

Reply:
이거에대해 고려해봤는데 스펙문서상에서도 "복제 테이블" 이라는 표현을 썼어서 일단은 유지하려합니다.